cpp/src/parquet/column_reader.cc

```diff
@@ -130,13 +130,13 @@ class SerializedPageReader : public PageReader {
       column_has_dictionary_ = ctx->column_has_dictionary;
       row_group_ordinal_ = ctx->row_group_ordinal;
       column_ordinal_ = ctx->column_ordinal;
-#ifdef PARQUET_ENCRYPTION
       meta_decryptor_ = ctx->meta_decryptor;
       data_decryptor_ = ctx->data_decryptor;
       if (data_decryptor_ != NULLPTR || meta_decryptor_ != NULLPTR) {
+#ifdef PARQUET_ENCRYPTION
```

Comment:
Hi Tham, I believe we need to try to get rid of all #ifdef PARQUET_ENCRYPTION statements (if possible). Lets go over the code early next week (our weekend starts today..).